digdag-core/src/main/java/io/digdag/core/database/AESGCMSecretCrypto.java

```diff
@@ -92,7 +93,11 @@ public String encryptSecret(String plainText)
             throw new IllegalArgumentException("Too long text");
         }
 
-        byte[] recordBytes = new byte[RECORD_SIZE];
+        int recordContentsLength = LENGTH_SIZE + plainTextBytes.length;
+
+        int recordLength = RECORD_SIZE_ALIGNMENT * ((recordContentsLength + RECORD_SIZE_ALIGNMENT - 1) / RECORD_SIZE_ALIGNMENT) + GCM_TAG_LENGTH;
```

Comment:
Why do we need to add `GCM_TAG_LENGTH` here? `recordBytes` doesn't seem to include any tag field. `cipher.doFinal(recordBytes)` appends a tag implicitly?